scripts/get_freqs_bins.py

```diff
@@ -0,0 +1,45 @@
+from typing import List
+import numpy as np
+import torch
+
+# monkey patch Path
+from fastai.vision import *
+
+from artificial_contrast.data import get_scans
+from artificial_contrast.dicom import read_HU_array
+from artificial_contrast.freqs import freqhist_bins, hist_scaled_img
+from artificial_contrast.settings import SEED
+
+
+DATA_PATH = Path(os.environ['DATA'])
+FREQS = os.environ.get('FREQS', 'freqs.npy')
+WINDOWS = [-100, 300]
+
+
+def remove_outside_window(
+    tensor: torch.Tensor, windows: List[int] = WINDOWS
+) -> torch.Tensor:
+    min_val, max_val = windows
+    tensor = tensor[(tensor >= min_val) & (tensor <= max_val)]
+    return tensor
+
+
+scans = get_scans(DATA_PATH, test=False)
+sample_dcms = torch.stack(
```

Comment:
It looks a bit combo stack + tuple + list, but I assume that's the simplest way to achieve desired structure